tests/DotNetLightning.Core.Tests/CommitmentToLocalExtensionTests.fs

```diff
@@ -0,0 +1,38 @@
+module CommitmentToLocalExtensionTests
+
+open System
+open NBitcoin
+open Expecto
+open DotNetLightning.Utils
+open DotNetLightning.Transactions
+open DotNetLightning.Channel
+
+open ResultUtils
+open ResultUtils.Portability
+
+[<Tests>]
+let commitmentToLocalExtensionTests =
+    testList "CommitmentToLocalExtensionTests" [
+        testCase "can extract parameters" <| fun _ ->
+            let rand = new Random()
```

Comment:
nit: unnecessary `new`, `new` keyword in F# should be always coupled to an usage of `IDispoasble` , thus you should use `use` instead of `let` .So when you instantiate `Key` you should `use` it.

Reply:
Fixed.